src/SonarScanner.MSBuild.Common/MsBuildPathSettings.cs

```diff
@@ -170,7 +182,10 @@ public IEnumerable<string> GetGlobalTargetsPaths()
 
             return new[]
             {
+                // Up to v15, global targets are dropped under Program Files (x86)\MSBuild.
+                // This doesn't appear to be the case for later versions.
```

Comment:
Miscellaneous: we emit a warning if we detect a targets file under _Program Files_, but that's all. 
We should have been checking under 14.0 and 15.0 but weren't, so I've fixed that too.
I tried adding a target under "Current" to see if it was picked up but it wasn't, so I guess it isn't required for MSBuild 16.